docker/coc-member/.env.local.backend.example

```
@@ -0,0 +1,31 @@
+REFRESH_TTL=7
+
+# Rate limiting
+RATE_LIMIT_WINDOW_MINUTES=1
+RATE_LIMIT_MAX_REQUESTS=1000
+
+# CORS — set to your frontend's origin
+ALLOWED_ORIGINS=http://localhost:5173
+
+# Email service (Resend)
+EMAIL_ID=your_sender@example.com
+CONTACT_EMAIL_ID=callofcode07@gmail.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONTACT_EMAIL_ID key should go before the EMAIL_ID key

(UnorderedKey)